backend/auth/login.js

```diff
@@ -1,15 +1,23 @@
 const express = require('express');
 require('dotenv').config();
+const { body, validationResult } = require('express-validator');
 const authService = require('./services/authService');
 
 const router = express.Router();
 
-router.post('/', async (req, res) => {
+router.post('/', body('email').isEmail().normalizeEmail(), async (req, res) => {
```

Comment:
what is '.isEmail().normalizeEmail()'?
I dont like it. What if your body have a lot of fields which need to be transformed? 
It's better to create functions for each entity which will do this work